### ISSUE TITLE
Telegram and SMS messages with images don't trigger API

### ISSUE THREAD

**Issue by jmatthewpryor** [phonetonote/phone-to-roam-client#16]
Hi

I have a build of the Roam client running locally and the `fetchNotes()` function doesn't seem to fetch notes that have images attached.

I added some console logging and I can see that 
`  axios(`${SERVER_URL}/feed.json?roam_key=${roamKey}`)`
returns items when they message is plan text but not that I send images& text via SMS or Telegram

**Comment by scottrblock** [comment]
Hey @jmatthewpryor looking at the twilio logs, I don't see any messages from your number with media attached.

I haven't configured attachments to work with telegram messages. I would need to figure out some storage stuff first i think. But images from sms should work.

**Comment by scottrblock** [comment]
I just re-tested an sms image and it seems to be working. @jmatthewpryor you're sending messages from your verified phone number to the same phone number as normal?

**Comment by jmatthewpryor** [comment]
yes - AFAIR I tested SMS & Telegram - text based messages work on both - I will retest SMS from my registered number
